samples/dotnet-runtimeinfo/dotnet-runtimeinfo.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Displays .NET version and environment information.</Description>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <LangVersion>9</LangVersion>
```

Comment:
Should we update this to `10` or `latest`?

Reply:
sure, done.